src/elm/Cambiatus/Object/Community.elm

```diff
@@ -24,11 +24,6 @@ actionCount =
     Object.selectionForField "Int" "actionCount" [] Decode.int
 
 
-hasActions : SelectionSet Bool Cambiatus.Object.Community
-hasActions =
```

Comment:
weird, the latest version of the backend has this, why the generator removed it?

Reply:
I removed it manually. For some reason when merging master it was declared twice.

Reply:
I guess that's because it was already generated when Payment History was merged (here's the related [discussion](https://github.com/cambiatus/frontend/pull/253#discussion_r438650208)). Anyway, generating new graphql files does not bring any changes, so everything looks good with this.